### DISCUSSION TITLE
Question: Can I redistribute `Microsoft.Windows.SDK.NET.dll` and `WinRT.Runtime.dll`?

### DISCUSSION THREAD

**Discussion by yokra9** [microsoft/WindowsAppSDK#4368] [Q&A] [answered]
Hello! I created a WPF project and configured it for Windows App SDK support without any NuGet packages. I used the below article as a guide.
<https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/wpf-plus-winappsdk>

After building the project, I found 2 Dlls in the artifact directory:

* `Microsoft.Windows.SDK.NET.dll`: Windows SDK for .Net 6 (seems this repo)
* `WinRT.Runtime.dll`: C#/WinRT Runtime

However, I cannot find the license file for the libraries. Can I redistribute these with my app?

**Comment by hez2010** [comment]
They're from this repo: https://github.com/microsoft/CsWinRT, which are licensed under MIT as well.

**Comment by manodasanW** [comment]
You can find the license file for those binaries under License Info on our [NuGet page](https://www.nuget.org/packages/Microsoft.windows.sdk.net.ref).

**Comment by yokra9** [comment]
Thanks for the early response! I understood the following:

| DLL | Repository | License |
| - | - | - |
| `Microsoft.Windows.SDK.NET.dll` | [Windows App SDK](https://github.com/microsoft/WindowsAppSDK) | [Microsoft Software License Terms<br>Microsoft Windows Software Development Kit (SDK) for Windows 10](https://aka.ms/WinSDKLicenseURL) |
| `WinRT.Runtime.dll` | [C#/WinRT Runtime](https://github.com/microsoft/CsWinRT) | [The MIT License](https://opensource.org/license/mit) |

However, This repo seems to [be under MIT](https://github.com/microsoft/WindowsAppSDK/blob/main/LICENSE). Are there any points to consider compared to other projects under MIT?

maybe related: #4197

**Comment by manodasanW** [comment, marked as answer]
I am personally not a legal expert to comment on specific licensing terms.  But I do want to note that `Microsoft.Windows.SDK.NET.dll` is not part of the Windows App SDK that comes from this repo.  It is instead a C# projection for the [Windows Platform SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-sdk/), that Windows App SDK builds on top of, and uses the same licensing terms as the other [Windows SDK NuGet packages](https://www.nuget.org/profiles/WindowsSDK).

**Comment by yokra9** [comment]
I finally recognized both of these DLLs are from CsWinRT and not a part of this repo. Thank you very much for answering my out-of-place question!
